GoodMedicalApp/Controllers/TypeOperationController.cs

```diff
@@ -52,5 +52,26 @@ public ContentResult Remove(int id)
                 return Content("<p>The operation's type wasn't found!</p>");
             }
         }
+
+        public ActionResult FormUpdate(int id)
+        {
+            var typeOperation = typeOperationService.GetItemById(id);
+            if (typeOperation != null)
+            {
+                return View(typeOperation);
+            }
+            else
+            {
+                return Content("<p>The operation's type wasn't found!</p>");
+            }
+        }
+
+        [HttpPost]
+        public ContentResult Update(TypeOperation typeOperation)
+        {
+            typeOperationService.Update(typeOperation);
+            typeOperationService.Save();
```

Comment:
As I think, it will be better to include Save into service, instead of controller.
Controller doesn't has to know if we need to perform db actions or not